tests/test_sns_filters.py

```diff
@@ -45,6 +45,41 @@ def test_peak_offset_filter(self):
         peak_offset_filter(results, peak_offset_max=1)
         self.assertEqual(4, len(results))
 
+    def test_predictive_line_cluster(self):
+        num_times = 10
+        height = 40
+        width = 50
+        times = np.arange(num_times) + 60676  # MJD for Jan 1, 2025
+
+        # Create a fake data set a few fake objects with different fluxes.
+        ds = FakeDataSet(width=width, height=height, times=times, use_seed=11, psf_val=1e-6)
+
+        trj = ds.insert_random_object(5)
+        for it in np.arange(5):
+            ds.trajectories.append(trj)
+        results = ds.make_results()
+
+        self.assertEqual(6, len(results))  # 6 total trajectories inserted
+        predictive_line_cluster(results, times)
+        # All 6 trajectories are the same so expect them to be clustered into 1
+        self.assertEqual(1, len(results))
+
+        ds = FakeDataSet(width=width, height=height, times=times, use_seed=11, psf_val=1e-6)
+        ds.insert_object(trj)
```

Comment:
we should do the fake object insertion here, to make sure the two test cases aren't contaminating each other